packages/form/src/SelectRow.js

```diff
@@ -200,6 +200,7 @@ class SelectRow extends PureComponent {
           const valueMap = valueLabelMap.get(value) || {};
           return valueMap.label;
         })
+        .filter(label => Boolean(label))
```

Comment:
也可以直接這樣，跟 @YuCJ  學到的 XD
```suggestion
        .filter(Boolean)
```

Reply:
也是可以但稍微要想一下xd